### ISSUE TITLE
[Bug]: Consumption planning assumes deviation will stay "forever"

### ISSUE THREAD

**Issue by SimonYoungtree** [victronenergy/dynamic-ess#95]
### Contact Details

siempoot@gmail.com

### VRM portal ID

c0619ab31cc4

### Country / region

Netherlands (nl)

### B max

40

### TB max

6

### FB max

6

### TG max

16

### FG max

16

### Battery costs

0.027

### Buy price

(p+0.13)*1.21+0.02

### Sell price

(p+0.13)*1.21-0.02

### feed-in possible

yes

### feed-in control

None

### Version

0.1.6

### What happened?

Sometimes, when there is substantial more consumption than planned, a recalculation of future consumption assumes the deviation will last until the foreseeable future. This harms future SOC planning. I know that at some point in time during the day, when a new recalculation is done, this will be corrected, but some harm may have been done.

I do  not know how you do the consumption planning, but if a deviation of the planned the consumption is found, then only assume this will last until the next planning recalculation.

This is not an urgent matter. Please go on with fixing other bugs. You are doing a great job.

![image](https://github.com/victronenergy/dynamic-ess/assets/139110412/ffee837f-81e2-4b0f-a8a8-65425ce195bc)

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

**Comment by arrow1800** [comment]
yes this is an urgent manner! at least for me it is. I see this behavior when when my EV charger turns on. (load of 11kw for a few hours, which is mostly finished at the end of the afternoon) the whole evening DESS is forecasting huge consumptions and therefor unnecessarily (dis)charging the battery. 

is there a way to force recalculation? or to ignore load for a certain time period?
